packages/web-frontend/src/components/View/ChartWrapper/ReferenceLabel.js

```diff
@@ -7,14 +7,21 @@
 
 import React from 'react';
 
+function returnTrueIfMax(value) {
+  if (value.includes('MAX') || value.includes('Max') || value.includes('max')) return true;
```

Comment:
Possibly you could use `value.toLowerCase().includes('max')`?
![image](https://user-images.githubusercontent.com/32168886/100966605-7383c180-3581-11eb-8e66-69fb8298e67a.png)

Reply:
Also if you want to be super fancy this whole function could be one line:
```
const isMaxLabel = value => value.toLowerCase().includes('max');
```